data/systemd/snapd.manage

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+#
+# This script will start snapd from a base snap. This script knows
```

Comment:
```
This script acts as an entry point for running snapd from snapd.snap on
a base snap that otherwise doesn't ship snapd itself. The entry point
arranges for appropriate system integration, in particular by creating
and starting systemd units for running snapd without any confinement.
```